src/data-generator/users.ts

```diff
@@ -311,19 +317,31 @@ export async function createUsers(
     )
   }
   log('Field agents created')
-  log('Creating', config.hospitalFieldAgents, 'hospitals')
-  for (let i = 0; i < config.hospitalFieldAgents; i++) {
-    const systemAdmin =
-      systemAdmins[0] ||
-      (await createUser(token, randomOffice.id, countryCode, phoneNumberRegex, {
-        role: 'LOCAL_SYSTEM_ADMIN',
+
+  log('Creating natlSysAdmin user')
+  for (let i = natlSystemAdminUser.length; i < config.natlSysAdmin; i++) {
+    natlSystemAdminUser.push(
+      await createUser(token, randomOffice.id, countryCode, phoneNumberRegex, {
+        role: 'NATIONAL_SYSTEM_ADMIN',
         type: ''
-      }))
+      })
+    )
+  }
+  log('NatlSystemAdmin user created')
+
+  log('Creating', config.natlSysAdmin, 'hospitals')
+  for (let i = 0; i < config.natlSysAdmin; i++) {
+    const natlSystemAdmin =
+    natlSystemAdminUser[0] ||
+    (await createUser(token, randomOffice.id, countryCode, phoneNumberRegex, {
+      role: 'NATIONAL_SYSTEM_ADMIN',
+      type: ''
+    }))
```

Comment:
We should do it how it was previously done. Create `config.hospitalFieldAgents` no of hospitals and not `config.natlSysAdmin` times. We don't even need the `config.natlSysAdmin` as mentioned in a earlier comment